databricks/koalas/plot.py

```diff
@@ -90,7 +90,7 @@ def get_sampled(self, data):
             sampled = data._sdf.sample(fraction=float(self.fraction))
             return DataFrame(data._internal.copy(sdf=sampled)).to_pandas()
         elif isinstance(data, Series):
-            scol = data._kdf._internal.data_scols[0]
+            scol = data._scol
```

Comment:
BTW, the issue about it was, for instance,

```python
kdf = ks.range(10)
kdf.id = kdf.id - 1
```

In this case, `data._scol` will become `Column<id> - 1` but `_kdf._internal.data_scols[0]` will become just column `Column<id>`.

Reply:
ahh, get it, good tip!! thanks, man! @HyukjinKwon